src/main/index.ts

```diff
@@ -236,8 +239,10 @@ app.on("ready", async () => {
   logger.info("🖥️  Starting WindowManager");
   const windowManager = WindowManager.createInstance();
 
+  const menuItems = di.inject(electronMenuItemsInjectable);
+
   onQuitCleanup.push(
-    initMenu(windowManager),
+    initMenu(windowManager, menuItems),
```

Comment:
This would be cleaner if function `initMenu` was `injectable`. @panuhorsmalahti would you like to do this with us?

Reply:
I guess, what is the benefit of this being injectable here? Instead of just computing the entries?

Reply:
This file shouldn’t need to know what are the dependencies for `initMenu`. Simplicity.

Reply:
I meant, why is `menuItems` not just a computed without this injection

Reply:
Same argument:

`menuItems` has dependencies that I shouldn’t need to know when I use it. 

All this code should care is that the required interface is somehow fulfilled.

Reply:
@jansav I'm afraid I'm a bit busy with other Lens 🔥 issues for the time being..

Reply:
No worries @panuhorsmalahti :)

We can share you the result, if you want.

Reply:
Sure, that sounds interesting.

Reply:
I do believe the promised result will satisfy @Nokel81, as it will extract responsibilities related to instantiation away from this file, where it doesn't quite belong. @Nokel81 how about you, wanna have a go doing this with us? It may be trivial here, but having this freedom somewhere else will be a big deal. 

Or the beer's on me ;)

Reply:
freedom?